src/pages/Profile.jsx

```diff
@@ -0,0 +1,39 @@
+// ApplicationForm.js
+import React from 'react';
+// import { Link }  from 'react-router-dom';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+// Signup.js
+import Signin from '../components/Signin';
+import { Container } from 'react-bootstrap';
+import Signup from '../components/Signup';
+import { AuthProvider } from '../contexts/AuthContext';
+
+const Profile = (props) => {
+    return (
+      <>
+          <h1>Profile</h1>
+  
+        <AuthProvider>  
+          <Container 
+            className='d-flex align-items-center justify-content-center'
+            style={{ minHeight: "80vh"}}
+          >
+            
+            <div id = "student-info">
+            <img src = ""></img>
```

Comment:
Good job on noticing the detail on Figma, of reserving space for the image for student's photo.